### PR TITLE
Ratification Polls for March 2023

### PR DESCRIPTION
**Pull request by blimpa**
- Constitution MIP Set + MIP102c2-SP1
- MIP41c4-SP40: Onboarding Retro as CES-001 Facilitator
- MIP14c2-SP5: Requesting Protocol DAI Transfer to fund TechOps CU for March 2023
- MIP39c2-SP39: Modifying Sidestream Auction Services Core Unit Mandate - SAS-001
- MIP40c3-SP95: Modifying Sidestream Auction Services Core Unit Budget (SAS-001)
- MIP14c2-SP4: Starknet CU Budget Extension (SNE-001)

@patrick-j-govalpha , @prose11 , @Gala-G

### CODE REVIEW

File: governance/polls/Budget Ratification Poll for Modifying Sidestream Auction Services Core Unit Budget (SAS-001) (MIP40c3-SP95) - March 13, 2023.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+summary: MIP40c3-SP95 modifies the DAI budget for the SAS-001 Core Unit, continuing operations from 2023-04-01 until 2024-03-31.
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp95-modifying-sidestream-auction-services-core-unit-budget-sas-001/19737
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: $YYYY-MM-DDT16:00:00
```
Comment (Gala-G) [inline governance/polls/Budget Ratification Poll for Modifying Sidestream Auction Services Core Unit Budget (SAS-001) (MIP40c3-SP95) - March 13, 2023.md:12]: change dates

File: governance/polls/Budget Ratification Poll for Modifying Sidestream Auction Services Core Unit Budget (SAS-001) (MIP40c3-SP95) - March 13, 2023.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+The Governance Facilitators have placed a budget ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, March 13 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to $mip_code being accepted and implemented in the Maker Protocol.
```
Comment (Gala-G) [inline governance/polls/Budget Ratification Poll for Modifying Sidestream Auction Services Core Unit Budget (SAS-001) (MIP40c3-SP95) - March 13, 2023.md:26]: MIP40c3-SP95

File: governance/polls/Ratification Poll for Modifying Sidestream Auction Services Core Unit Mandate - SAS-001 (MIP39c2-SP39) - March 13, 2023.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, March 13 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to $mip_code being accepted and implemented in the Maker Protocol.
```
Comment (Gala-G) [inline governance/polls/Ratification Poll for Modifying Sidestream Auction Services Core Unit Mandate - SAS-001 (MIP39c2-SP39) - March 13, 2023.md:26]: MIP39c2-SP39

File: governance/polls/Ratification Poll for Onboarding Retro as CES-001 Facilitator (MIP41c4-SP40) - March 13, 2023.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, March 13 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to $mip_code being accepted and implemented in the Maker Protocol.
```
Comment (Gala-G) [inline governance/polls/Ratification Poll for Onboarding Retro as CES-001 Facilitator (MIP41c4-SP40) - March 13, 2023.md:26]: MIP41c4-SP40

File: governance/polls/Ratification Poll for Requesting Protocol DAI Transfer to fund TechOps CU for March 2023 (MIP14c2-SP5) - March 13, 2023.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, March 13 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to $mip_code being accepted and implemented in the Maker Protocol.
```
Comment (Gala-G) [inline governance/polls/Ratification Poll for Requesting Protocol DAI Transfer to fund TechOps CU for March 2023 (MIP14c2-SP5) - March 13, 2023.md:26]: MIP14c2-SP5

File: governance/polls/Ratification Poll for Starknet CU Budget Extension (SNE-001) (MIP14c2-SP4) - March 13, 2023.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, March 13 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to $mip_code being accepted and implemented in the Maker Protocol.
```
Comment (Gala-G) [inline governance/polls/Ratification Poll for Starknet CU Budget Extension (SNE-001) (MIP14c2-SP4) - March 13, 2023.md:26]: MIP14c2-SP4

File: governance/polls/Ratification Poll for the Maker Constitution MIP Set and MIP102c2-SP1 (multiple MIPs) - March 13, 2023.md [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, March 13 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to $mip_set being accepted and implemented in the Maker Protocol.
```
Comment (Gala-G) [inline governance/polls/Ratification Poll for the Maker Constitution MIP Set and MIP102c2-SP1 (multiple MIPs) - March 13, 2023.md:26]: Constitution MIP Set

Comment (Gala-G) [reply governance/polls/Ratification Poll for the Maker Constitution MIP Set and MIP102c2-SP1 (multiple MIPs) - March 13, 2023.md:26]: Maker Constitution MIP Set and MIP102c2-SP1 (multiple MIPs)

**Review by Gala-G — COMMENTED** [review]
Left some comments @blimpa

**Comment by blimpa** [comment]
> Left some comments @blimpa 

Haha. Thank you!

**Comment by patrick-j-govalpha** [comment]
@blimpa dumb thing but it's "Constitution MIP Set" not "Maker Constitution MIP Set"

**Pull request merged**